src/Ting/Repository/HydratorSingleObject.php

```diff
@@ -34,6 +34,11 @@
 class HydratorSingleObject extends Hydrator
 {
 
+    public function __construct()
+    {
+        parent::__construct();
```

Comment:
Why override the construct to only call the parent?

Reply:
It's indeed overkill :)